### ISSUE TITLE
Issue on the tutorial

### ISSUE THREAD

**Issue by HuangHam** [neurora/NeuroRA#5]
I think there is an issue on section 5 (Section 5: Calculating the RDM for ROI and Plotting) of the second Tutorial (fMRI one). The output of that cell is RDMs computing finished! Invalid input!

**Comment by HuangHam** [comment]
Oh I figured out why. so in the plotting line instead of plot_rdm(rdm_roi ...., you should do plot_rdm(rdm_roi[0] .... to index the first subject's RDM.

**Comment by NickGuAI** [comment]
I noticed the example provides plotting only for searchlight RDM
were you able to figure out how to plot roi rdm?

**Comment by HuangHam** [comment]
yes. So as I said in the previous comment, the rdm_roi output by the tutorial code is a list of rdm where each element is the rdm for on subject. You can either plot the rdm of one subject using plot_rdm(rdm_roi[subj_indx],...) or you can average all rdm across all subjects.

**Comment by NickGuAI** [comment]
thanks for the response @HuangHam !
Sorry I meant in Section 8: Saving the RSA result and Plotting

which plots searchlight only. I was wondering if you figured out a way to plot for non-searchlight rsa results

**Comment by HuangHam** [comment]
Oh I see. I did not look at Section 8 closely, but it seems like that it's plotting the brain regions where the searchlight RSA shows a significant effect. In case of ROI, you already prespecify a brain region, so I don't see why section 8 would apply... Maybe that's why the tutorial only includes the demo for plotting searchlight.

**Comment by NickGuAI** [comment]
ah *derp
that's indeed the case

Thanks @HuangHam !!
